roles/foreman_repositories/tasks/main.yml

```diff
@@ -1,12 +1,5 @@
 ---
-- include_tasks: debian_release_repos.yml
-  when: ansible_os_family == 'Debian' and foreman_repositories_use_release
-
-- include_tasks: redhat_release_repos.yml
-  when: ansible_os_family == 'RedHat' and foreman_repositories_use_release and not foreman_repositories_use_koji
-
-- include_tasks: koji_repos.yml
-  when: foreman_repositories_use_koji
+- include_tasks: "{{ ansible_os_family|lower }}_{{ foreman_repositories_flavor }}_repos.yml"
```

Comment:
I don't guess there is a 'el' or 'enterprise linux' "family" ? 'redhat' sounds specific when you are considering there is an RH product based on these projects.

Reply:
Yes, it's the same with Debian (osfamily) vs Debian (distro). There the situation is easier because they use dist release names which are unique. This doesn't handle Fedora but it didn't do so before either. This is IMHO where something like Hiera could be nice and I'd gladly hear of better patterns to solve multi-distro tasks. It's something I think is the weakest point of Ansible.